Monika After Story/game/script-topics.rpy

```diff
@@ -4978,6 +4977,418 @@ label monika_beach:
 #    m 1k "That's why I love you, [player]!"
 #    return
 
+default persistent._mas_timeconcern = 0
+default persistent._mas_timeconcerngraveyard = False
+default persistent._mas_timeconcernclose = True
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_timeconcern",category=['Advice'],prompt="Sleep Concern",random=True))
+    rules = dict()
+    rules.update(MASSelectiveRepeatRule.create_rule(hours =range(0,5)))
+    rules.update({"monika wants this first":""})
+    addEvent(Event(persistent.greeting_database,eventlabel="monika_timeconcern",unlocked=False, rules=rules),eventdb=evhand.greeting_database)
```

Comment:
After talking with Potato, we decided that these should be refactored to separated greetings defined on `script-greetings.rpy` with a jump to the respective label. This is to have all greetings on the same place and make it easier to maintain.